client.go

```diff
@@ -64,6 +70,14 @@ func (c *Client) userAgent() string {
 	return defaultUserAgent
 }
 
+func (c *Client) timeout() time.Duration {
+	// return timeout in duration in seconds
+	if c.Timeout > 0 {
+		return time.Duration(c.Timeout) * time.Second
+	}
+	return time.Duration(defaultTimeout) * time.Second
```

Comment:
Can we return once and assign the timeout to a variable here? if/else?

Reply:
Just following the same syntax as the other methods (i.e. `userAgent()`)

Reply:
Resolved in [5b9c631](https://github.com/EasyPost/easypost-go/pull/47/commits/5b9c63119f3019cfb551a7fe991a4a88d9d16586)